test/Test/Fixtures.hs

```diff
@@ -206,3 +226,141 @@ issuesPending =
 
 attributionReportAsSerializedJson :: Text
 attributionReportAsSerializedJson = "{\"TestReport\": \"TestReportData\"}"
+
+signedUrl :: API.SignedURL
+signedUrl = API.SignedURL{API.signedURL = "https://foo.com"}
+
+-- Test data for licenseScanSourceUnits tests
+firstVendoredDep :: VendoredDependency
+firstVendoredDep =
+  VendoredDependency
+    "first-archive-test"
+    "vendor/foo"
+    (Just "0.0.1")
+
+secondVendoredDep :: VendoredDependency
+secondVendoredDep =
+  VendoredDependency
+    "second-archive-test"
+    "vendor/bar"
+    (Just "0.0.1")
+
+vendoredDeps :: NonEmpty VendoredDependency
+vendoredDeps = firstVendoredDep :| [secondVendoredDep]
+
+firstLocator :: Locator
+firstLocator =
+  Locator
+    "archive"
+    "42/first-archive-test"
+    (Just "0.0.1")
+
+secondLocator :: Locator
+secondLocator =
+  Locator
+    "archive"
+    "42/second-archive-test"
+    (Just "0.0.1")
+
+firstRevision :: RevisionInfo
+firstRevision =
+  RevisionInfo
+    "archive+42/first-archive-test$0.0.1"
+    True
+
+secondRevision :: RevisionInfo
+secondRevision =
+  RevisionInfo
+    "archive+42/second-archive-test$0.0.1"
+    True
+
+firstArchive :: Archive
+firstArchive =
+  Archive
+    "first-archive-test"
+    "0.0.1"
+
+secondArchive :: Archive
+secondArchive =
+  Archive
+    "second-archive-test"
+    "0.0.1"
+
+archives :: [Archive]
+archives = [firstArchive, secondArchive]
+
+locators :: NonEmpty Locator
+locators = firstLocator :| [secondLocator]
+
+firstLicenseSourceUnit :: LicenseSourceUnit
```

Comment:
`firstLicenseSourceUnit` and `secondLicenseSourceUnit` will become much smaller once we can ignore the request sent to a test